<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelAbout</class>
 <widget class="QDialog" name="PanelAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Oxymetry</string>
  </property>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Oxymetry 0.7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_compiler">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&lt;Compiler info&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_build">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Build info&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_copyright">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Copyright info&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_license">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;License&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>451</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Changelog</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit_changelog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>425</width>
       <height>135</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Version 0.7: (15.06.2018)
+ Zoom of Average signal plot, now the user is able to zoom in the plots.
+ New interface to hide/show plots.
+ The user can now Normalize the measurements in counts per second.
+ New interface for the configuration of the measurements.
+ The configuration information is now saved into a file, which allows the user to load a complete measurement setting.
+ The user can now modify the configuration file during the calibration phase.
+ Now the plots are divided in tabs, which will allow a separation between phases of the measurement.
+ New user interface for the selection of files for validation and calibration called &quot;Analize Data&quot;.
+ More user friendly status bar.
+ The configuration interface now shows more precisely the required time for a measurement, as well as the time of start and end.
+ The user can select a delay in the start of the measurement in the configuration, for example 8 hours of waiting until the measurement starts.
+ The user can change the size of the columns in the Measurement List in order to read the file names more easily.
+ The user can create up to 2 impurities plus the glucose concentration.
+ Loading old data now shows only the needed plots.
+ Now the user has a dialog counting down 5 seconds before the start of the measurements.
+ The average plot shows numerically the average value of the signals.
+ The Maximum DC intensity and it's wavelength are now shown in the FFT plots in the side panel.
- Plot of Concentration prediction in main plots window. It's now present in a different plots tab.
* Several bugs fixed during calibration, measurement and configuration.

Version 0.6.1: (17.01.2018)

+ Plots and GUI elements selection.
+ GUI elements by demand.
+ Plot of Concentration prediction.
+ Update of GUI according to measurements.
+ Non Modal help window.
+ preview Tab now is also updated from Polarimeter tab.
+ Polarimeter plots are now a class.
- Zoom of Average signal plot.

Version 0.6: (22.12.2017)

+ Reintroduced the 32 bit version
+ Improved build process
+ Fluidic toolkit now supports roller pump driven by stepper motor
+ Tabs in panel are now handled as modules
+ Configuration file is now created automatically
+ Inverse Adding-Doubling algorithm is no external program anymore, but integrated as library instead
* Several bug fixes at automatic adjustment of integration time

Version 0.5: (05.12.2017)

Polarimeter Tab:
+ Live FFT in periods of time depending on spectrometer settings.
+ Live Raw signal and live FTT.
+ Live Raw signal during breaks of measurements.
+ Quit oxymetry now closes all running loops and closes the app completely.
+ Information Bar at the botton.
+ Toggle Button for data loading.
+ Clear All option to restart all configurations and plots.
+ Auto change of name of files in measurement profile when changes are made during calibration.
+ Calibration independent of measurement, but could be done with the loaded configuration.
+ Automatic adjustement of plots according to all new settings.
+ Autoadjustment of integration time adapted to calibration.
+ Warnings on adjustement of settings out of range.
+ Auto adjusting of frequency according to selected integration time when needed. As well as a top limit.
- Live Average Raw Signal Display with time dependency. (Not needed).
+ Help button to guide user through the polarimeter tab.
+ When saving data, the old data with thte same name isn't overwrited, instead a new folder date_X is created, where X is a number.
+ When changing Nr Spectra or Frequency values but not pressing enter, the old values are returned to the table cell.
+ Automatic adjustement of axis on Live Raw Signal plot.

Version 0.4.3: (26.10.2017)

Polarimeter Tab:
+ Live Average Raw Signal Display with time dependency.
+ Autoadjust of Plot X and Y axis according to the ploted data.
+ Load of data from FFT file or Raw Data File.
+ Save option of FFT Analyzed Data.
+ Save Graphs to PDF.
+ Measurement Profile Load or Edition.
+ Calibration Option for Polarimeter Tab.
+ Plot of the FFT at the selected wavelength.
+ Raw Signal Plot independent from Preview Tab.
+ Spectra, Integration Time and Averaging Profile are now shown in Measurement Profile.
+ Colored Start/Stop measurement toggle button.
- Disabled preview/polarimeter setup when one preview/polarimeter setup is active.

Version 0.4.2: (08.10.2017)

- Dropped support for 32-bit OS
+ Code cleanup and restructuring

Version 0.4.1: (06.10.2017)

+ Added button for polarimeter setup
+ Implemented FFT data analysis
+ Disabled preview/polarimeter setup when one preview/polarimeter setup is active
+ Modification of polarimeter setup tab panel
+ Toggle button for polarimeter setup
+ File and folder creation when polarimetric measurement starts

Version 0.4.0: (02.10.2017)

+ Added files for polarimeter control
+ Added FFTW library and data folder

Version 0.3.0: (29.09.2017)

* Legend now shows 'unknown' if serial number or readable name of spectrometer are not available
* Fixed spectra count in StoreToRAM files
* Changes of readable name are now accepted
* Disabling/enabling of a device now disables all elements in device item
+ Many code optimizations, especially to the StoreToRAM function and the spectrometer configuration dialog
+ Time-controlled measurements can now be automated via button 'time pattern'
+ The serial number of the device is no longer attached to the file names of StoreToRAM
+ StoreToRAM now supports more than 10000 spectra
+ Wavelengths are shown belong pixel numbers in configuration dialog
+ Combined 'preview start' and 'preview stop' buttons into one single button

Version 0.2.6: (04.09.2017)

* Fixed a bug that prevented data export
+ Increased maximum of StoreToRAM spectra
+ StoreToRAM dialog remembers last path

Version 0.2.5: (14.08.2017)

* Updated AvaSpec library to version 9.6.0.0

Version 0.2.4: (19.06.2017)

+ Support of increased peristaltic pump RPM update rate
+ RPM in fluidic toolbox is now calculated over a period of one second
* Compatibility check for Qt 5.9.0

Version 0.2.3: (14.06.2017)

+ The user must now confirm whether existing files should be overwritten during data export

Version 0.2.2: (29.05.2017)

* Fixed calculation of fluidic circuit rollerpump RPM
* Fixed bug that prevented the first pinch valve from being switched
* Fixed a bug that caused the user to be overwhelmed with error messages when the light source was disconnected during operation
+ Added fluidic circuit auto-connect functionality
+ Added changelog and contributors lists
- Removed message if no spectrometers were found

Version 0.2.1 (24.05.2017):

+ The spectral data from the spectrometers are now stored in separate files
+ Additional information (dynamic dark correction enabled, time and date, ...) is now stored within the saved spectra

Version 0.2:

Initial release</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Contributors</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit_contributors">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>425</width>
       <height>135</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Benjamin Redmer, Cesar Andres Carvajal Arrieta, Torben Kruse, Isabeau Dibbern</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>label_license</zorder>
  <zorder>button_close</zorder>
  <zorder>label_title</zorder>
  <zorder>label_compiler</zorder>
  <zorder>label_build</zorder>
  <zorder>label_copyright</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
